actionpack/lib/action_dispatch/testing/integration.rb

```diff
@@ -675,6 +675,19 @@ def app=(app)
           @@app = app
         end
 
+        # Calling TestResponse#parsed_body on the response parses the response body based on the
+        # last response MIME type.
+        #
+        # Out of the box, only `:html` and `:json` are supported. But for any custom MIME
+        # types you've registered, you can add your own encoders with:
+        #
+        #   ActionDispatch::IntegrationTest.register_encoder :wibble,
+        #     param_encoder: -> params { params.to_wibble },
+        #     response_parser: -> body { Wibble.parse(body) }
+        #
+        # Where `param_encoder` defines how the params should be encoded and
+        # `response_parser` defines how the response body should be parsed through
+        # TestResponse#parsed_body.
```

Comment:
Let's tell what this method does, not what it is the behavior of `parsed_body`.

Something like `registers an encoder to be used to parses the response body returned by `parsed_body``